assets/js/components/Link.js

```diff
@@ -82,6 +87,7 @@ function Link( {
 				},
 			) }
 			href={ isAnchor ? href : undefined }
+			to={ isRouterLink ? to : undefined }
```

Comment:
Technically you can pass this one right through; if it's `undefined` it shouldn't be an issue; if it _isn't_ undefined we set the element as a React Router link that expects the "to" prop.

Reply:
Good point — passing straight through as it'll be `undefined` when expected.